### ISSUE TITLE
accessibility problems prevent me from getting to settings tab

### ISSUE THREAD

**Issue by zersiax** [Shippable/support#1393]
I am a screenreader user and currently unable to reach the settings -> Deployment keys portion of the settings area. The only settings I appear to be able to access are the general account settings. I need the deployment key for Heroku access, I cannot do work without it. Can this please be fixed, soon? Or at least give me a URL I can go to to reach this page directly? Thanks

**Comment by DavidGDragon** [comment]
We've redone the UI, see if you continue to have issues. Additionally, you can use this URL to get to your subscription page settings: `https://app.shippable.com/subscriptions/{{SUBSCRIPTION_ID}}/subscriptionsSettings`
